src/xrCore/FS.cpp

```diff
@@ -111,6 +114,7 @@ static int open_internal(LPCSTR fn, int& handle)
 #if defined(WINDOWS)
     return (_sopen_s(&handle, fn, _O_RDONLY | _O_BINARY, _SH_DENYNO, _S_IREAD));
 #elif defined(LINUX)
+    while (char* sep = strchr((char *)fn, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes the code more clear and allows to reduce code duplication.